docs/examples/FileExample/src/FileExample.ls

```diff
@@ -16,8 +16,8 @@ package
     public class FileExample extends Application
     {
 
-        // get a writable folder 
-        var writePath = Path.normalizePath(Path.getWritablePath() + "/Loom/Tests");
+        // get a writable folder
+        var writePath = Path.normalizePath(Path.getWritablePath() + "Loom/Tests");
```

Comment:
Yeah, it made an extra / in the path, it worked everywhere, but still better to have it fixed.